tests/cursor/cursor-IteratorIterator-004.phpt

```diff
@@ -2,7 +2,8 @@
 MongoDB\Driver\Cursor iteration beyond last document (OP_QUERY)
 --SKIPIF--
 <?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
-<?php NEEDS('STANDALONE_30'); ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_server_version('>', '3.0.99'); ?>
```

Comment:
Don't want to make these ``'>=', '3.1'`` ?

Reply:
That's technically cleaner. I'll do so.